0.16.0-release-notes.md

```diff
@@ -21,6 +21,9 @@ This is the draft release notes for 0.16.0. It is scheduled to be released on
 
 # Notable new features
 
+-   Command for creating temporary directories: path:temp-dir
+    ([#1255](https://b.elv.sh/1255)).
+
```

Comment:
Document path:temp-file too

Reply:
Done in the third commit of this P.R. which actually introduces that command.